src/pages/SearchPage.js

```diff
@@ -0,0 +1,81 @@
+import React, {useState} from 'react'
+import styled from 'styled-components'
+import { Link } from '@reach/router'
+import PetCard from './../components/PetCard'
+import LabeledSelect from './../components/common/LabeledSelect'
+import { petsDB, typeOpt, dogBreedsOpt, ageOpt, sizeOpt, genderOpt } from './../petsDB'
+
+const SearchPage = (props) => {
+    const [type, setType] = useState("Animal");
+    const [breeds, setBreeds] = useState([]);
+    const [age, setAge] = useState([]);
+    const [size, setSize] = useState([]);
+    const [gender, setGender] = useState();
+
+    const getStringValues = (value) => {
+        return value == null ? [] : value.map((item) => item.value);
+    }
```

Comment:
tight coupling with LabledSelect
are you sure that's the way to pass values around?
you've got nested calls in your onChange

Reply:
Fixed